src/main/java/io/github/dft/woocommerce/WooCommerceCustomers.java

```diff
@@ -0,0 +1,47 @@
+package io.github.dft.woocommerce;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.github.dft.woocommerce.model.customerapi.Customer;
+import io.github.dft.woocommerce.model.customerapi.CustomerWrapper;
+import lombok.SneakyThrows;
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.utils.URIBuilder;
+
+import java.net.http.HttpRequest;
+import java.util.HashMap;
+
+import static io.github.dft.woocommerce.constatndcode.ConstantCode.*;
+
+public class WooCommerceCustomers extends WooCommerceSdk{
+    public WooCommerceCustomers() {
+        super();
+    }
+
+    @SneakyThrows
+    public CustomerWrapper getAllCustomers(String storeDomain,HashMap<String, String> params){
+        URIBuilder uriBuilder = new URIBuilder(HTTPS.concat(storeDomain.concat(API_BASE_END_POINT
```

Comment:
Remove external dependencies use URI instead of URIBuilder ask to abhishek how to do that.

Reply:
.

Reply:
Ok